ledger/error/src/main/scala/com/daml/error/ErrorClass.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.error
+
+/** The classes [[ErrorClass]] and [[ErrorGroup]] are used to hierarchically structure error codes (their
+  * hierarchical structure affects how they are displayed on the website)
+  */
+case class ErrorClass(docNames: List[String]) {
```

Comment:
I would name this class e.g `ErrorGroup(Hierarchy)Path`  because:
1) AFAICT This class is encoding a path in a nesting of error groups. 
2) Having error categories, codes, groups and also classes (and what's the distinction) is a bit hard to wrap my head around quickly.
3) We avoid semantic class with JVM's notion of classes (e.g. when writing docs; what 'error class' refers to might be ambiguous)

Reply:
For avoiding breaking changes with dependents of this API, I would defer from major refactorings here. This is a good discussion point for a refactoring PR, though